### ISSUE TITLE
cash-global installation error

### ISSUE THREAD

**Issue by Jonarzz** [dthree/cash#10]
When I'm trying to install cash-global, I get such an error:

```
C:\>npm install cash-global -g

> cash-global@0.1.0 preinstall C:\Users\user\AppData\Roaming\npm\node_modules\cash-global
> node ./bin/preinstall.js

C:\Users\user\AppData\Roaming\npm\node_modules\cash-global\bin\preinstall.js:26
    const excluded = (exclusions.indexOf(command) > -1);
    ^^^^^
SyntaxError: Use of const in strict mode.
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files (x86)\\nodejs\\\\node.exe" "C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "cash-global" "-g"
npm ERR! node v0.12.10
npm ERR! npm  v2.14.9
npm ERR! code ELIFECYCLE

npm ERR! cash-global@0.1.0 preinstall: `node ./bin/preinstall.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the cash-global@0.1.0 preinstall script 'node ./bin/preinstall.js'.
npm ERR! This is most likely a problem with the cash-global package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node ./bin/preinstall.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls cash-global
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\npm-debug.log
```

Content of npm-debug.log:
```
0 info it worked if it ends with ok
1 verbose cli [ 'C:\\Program Files (x86)\\nodejs\\\\node.exe',
1 verbose cli   'C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'install',
1 verbose cli   'cash-global',
1 verbose cli   '-g' ]
2 info using npm@2.14.9
3 info using node@v0.12.10
4 verbose install initial load of C:\Users\user\AppData\Roaming\npm\package.json
5 verbose readDependencies loading dependencies from C:\Users\user\AppData\Roaming\npm\package.json
6 silly cache add args [ 'cash-global', null ]
7 verbose cache add spec cash-global
8 silly cache add parsed spec { raw: 'cash-global',
8 silly cache add   scope: null,
8 silly cache add   name: 'cash-global',
8 silly cache add   rawSpec: '',
8 silly cache add   spec: '*',
8 silly cache add   type: 'range' }
9 silly addNamed cash-global@*
10 verbose addNamed "*" is a valid semver range for cash-global
11 silly addNameRange { name: 'cash-global', range: '*', hasData: false }
12 silly mapToRegistry name cash-global
13 silly mapToRegistry using default registry
14 silly mapToRegistry registry https://registry.npmjs.org/
15 silly mapToRegistry uri https://registry.npmjs.org/cash-global
16 verbose addNameRange registry:https://registry.npmjs.org/cash-global not in flight; fetching
17 verbose get https://registry.npmjs.org/cash-global not expired, no request
18 silly addNameRange number 2 { name: 'cash-global', range: '*', hasData: true }
19 silly addNameRange versions [ 'cash-global',
19 silly addNameRange   [ '0.0.2',
19 silly addNameRange     '0.0.3',
19 silly addNameRange     '0.0.4',
19 silly addNameRange     '0.0.5',
19 silly addNameRange     '0.0.6',
19 silly addNameRange     '0.0.7',
19 silly addNameRange     '0.0.8',
19 silly addNameRange     '0.0.9',
19 silly addNameRange     '0.0.10',
19 silly addNameRange     '0.0.11',
19 silly addNameRange     '0.0.12',
19 silly addNameRange     '0.0.13',
19 silly addNameRange     '0.0.14',
19 silly addNameRange     '0.0.15',
19 silly addNameRange     '0.0.16',
19 silly addNameRange     '0.0.17',
19 silly addNameRange     '0.0.18',
19 silly addNameRange     '0.1.0' ] ]
20 silly addNamed cash-global@0.1.0
21 verbose addNamed "0.1.0" is a plain semver version for cash-global
22 silly cache afterAdd cash-global@0.1.0
23 verbose afterAdd C:\Users\user\AppData\Roaming\npm-cache\cash-global\0.1.0\package\package.json not in flight; writing
24 verbose afterAdd C:\Users\user\AppData\Roaming\npm-cache\cash-global\0.1.0\package\package.json written
25 silly install resolved [ { name: 'cash-global',
25 silly install resolved     version: '0.1.0',
25 silly install resolved     description: 'Cross platform Linux-style commands.',
25 silly install resolved     main: './lib/index.js',
25 silly install resolved     scripts:
25 silly install resolved      { preinstall: 'node ./bin/preinstall.js',
25 silly install resolved        postinstall: 'node ./bin/postinstall.js' },
25 silly install resolved     repository:
25 silly install resolved      { type: 'git',
25 silly install resolved        url: 'git+https://github.com/cashjs/cash-global.git' },
25 silly install resolved     keywords:
25 silly install resolved      [ 'cash',
25 silly install resolved        'terminal',
25 silly install resolved        'emulator',
25 silly install resolved        'cygwin',
25 silly install resolved        'cli',
25 silly install resolved        'windows',
25 silly install resolved        'linux',
25 silly install resolved        'unix',
25 silly install resolved        'posix',
25 silly install resolved        'bash',
25 silly install resolved        'tty',
25 silly install resolved        'util',
25 silly install resolved        'vorpal',
25 silly install resolved        'vorpal.js' ],
25 silly install resolved     author: { name: 'dthree' },
25 silly install resolved     license: 'MIT',
25 silly install resolved     bugs: { url: 'https://github.com/cashjs/cash-global/issues' },
25 silly install resolved     homepage: 'https://github.com/cashjs/cash-global#readme',
25 silly install resolved     devDependencies: { gulp: '^3.9.0', 'gulp-eslint': '^1.1.1' },
25 silly install resolved     bin:
25 silly install resolved      { '$': './node_modules/cash/bin/cash.js',
25 silly install resolved        cash: './node_modules/cash/bin/cash.js' },
25 silly install resolved     dependencies: { cash: '*' },
25 silly install resolved     engines: { node: '>= 0.11.16', iojs: '>= 1.0.0' },
25 silly install resolved     files: [ 'commands.json', 'dist', 'bin' ],
25 silly install resolved     gitHead: '769b2517bfc65b8fecf6d17de6b01b14b71eb345',
25 silly install resolved     _id: 'cash-global@0.1.0',
25 silly install resolved     _shasum: 'c15a184fcf3c9a262cbc66ca4f63ab8509c492c0',
25 silly install resolved     _from: 'cash-global@*',
25 silly install resolved     _npmVersion: '3.3.6',
25 silly install resolved     _nodeVersion: '5.0.0',
25 silly install resolved     _npmUser: { name: 'dthree', email: 'threedeecee@gmail.com' },
25 silly install resolved     dist:
25 silly install resolved      { shasum: 'c15a184fcf3c9a262cbc66ca4f63ab8509c492c0',
25 silly install resolved        tarball: 'http://registry.npmjs.org/cash-global/-/cash-global-0.1.0.tgz' },
25 silly install resolved     maintainers: [ [Object] ],
25 silly install resolved     _npmOperationalInternal:
25 silly install resolved      { host: 'packages-5-east.internal.npmjs.com',
25 silly install resolved        tmp: 'tmp/cash-global-0.1.0.tgz_1456026581272_0.7278439523652196' },
25 silly install resolved     directories: {},
25 silly install resolved     _resolved: 'https://registry.npmjs.org/cash-global/-/cash-global-0.1.0.tgz',
25 silly install resolved     readme: 'ERROR: No README data found!' } ]
26 info install cash-global@0.1.0 into C:\Users\user\AppData\Roaming\npm
27 info installOne cash-global@0.1.0
28 verbose installOne of cash-global to C:\Users\user\AppData\Roaming\npm not in flight; installing
29 verbose lock using C:\Users\user\AppData\Roaming\npm-cache\_locks\cash-global-c09d7c9a277aab3c.lock for C:\Users\user\AppData\Roaming\npm\node_modules\cash-global
30 silly install write writing cash-global 0.1.0 to C:\Users\user\AppData\Roaming\npm\node_modules\cash-global
31 verbose unbuild node_modules\cash-global
32 silly gentlyRm C:\Users\user\AppData\Roaming\npm\node_modules\cash-global is being purged from base C:\Users\user\AppData\Roaming\npm
33 verbose gentlyRm don't care about contents; nuking C:\Users\user\AppData\Roaming\npm\node_modules\cash-global
34 verbose tar unpack C:\Users\user\AppData\Roaming\npm-cache\cash-global\0.1.0\package.tgz
35 verbose tar unpacking to C:\Users\user\AppData\Roaming\npm\node_modules\cash-global
36 silly gentlyRm C:\Users\user\AppData\Roaming\npm\node_modules\cash-global is being purged
37 verbose gentlyRm don't care about contents; nuking C:\Users\user\AppData\Roaming\npm\node_modules\cash-global
38 silly gunzTarPerm modes [ '777', '666' ]
39 silly gunzTarPerm extractEntry package.json
40 silly gunzTarPerm modified mode [ 'package.json', 436, 438 ]
41 silly gunzTarPerm extractEntry README.md
42 silly gunzTarPerm modified mode [ 'README.md', 484, 502 ]
43 silly gunzTarPerm extractEntry bin/alias.js
44 silly gunzTarPerm modified mode [ 'bin/alias.js', 436, 438 ]
45 silly gunzTarPerm extractEntry bin/grep.js
46 silly gunzTarPerm modified mode [ 'bin/grep.js', 436, 438 ]
47 silly gunzTarPerm extractEntry bin/kill.js
48 silly gunzTarPerm modified mode [ 'bin/kill.js', 436, 438 ]
49 silly gunzTarPerm extractEntry bin/less.js
50 silly gunzTarPerm modified mode [ 'bin/less.js', 436, 438 ]
51 silly gunzTarPerm extractEntry bin/ls.js
52 silly gunzTarPerm modified mode [ 'bin/ls.js', 436, 438 ]
53 silly gunzTarPerm extractEntry bin/echo.js
54 silly gunzTarPerm modified mode [ 'bin/echo.js', 436, 438 ]
55 silly gunzTarPerm extractEntry bin/mv.js
56 silly gunzTarPerm modified mode [ 'bin/mv.js', 436, 438 ]
57 silly gunzTarPerm extractEntry bin/parser.js
58 silly gunzTarPerm modified mode [ 'bin/parser.js', 484, 502 ]
59 silly gunzTarPerm extractEntry bin/postinstall.js
60 silly gunzTarPerm modified mode [ 'bin/postinstall.js', 484, 502 ]
61 silly gunzTarPerm extractEntry bin/preinstall.js
62 silly gunzTarPerm modified mode [ 'bin/preinstall.js', 484, 502 ]
63 silly gunzTarPerm extractEntry bin/pwd.js
64 silly gunzTarPerm modified mode [ 'bin/pwd.js', 436, 438 ]
65 silly gunzTarPerm extractEntry bin/cp.js
66 silly gunzTarPerm modified mode [ 'bin/cp.js', 436, 438 ]
67 silly gunzTarPerm extractEntry bin/touch.js
68 silly gunzTarPerm modified mode [ 'bin/touch.js', 436, 438 ]
69 silly gunzTarPerm extractEntry bin/cd.js
70 silly gunzTarPerm modified mode [ 'bin/cd.js', 436, 438 ]
71 silly gunzTarPerm extractEntry bin/rm.js
72 silly gunzTarPerm modified mode [ 'bin/rm.js', 436, 438 ]
73 silly gunzTarPerm extractEntry bin/cat.js
74 silly gunzTarPerm modified mode [ 'bin/cat.js', 436, 438 ]
75 silly gunzTarPerm extractEntry bin/sort.js
76 silly gunzTarPerm modified mode [ 'bin/sort.js', 436, 438 ]
77 silly gunzTarPerm extractEntry bin/cash.js
78 silly gunzTarPerm modified mode [ 'bin/cash.js', 484, 502 ]
79 silly gunzTarPerm extractEntry bin/unalias.js
80 silly gunzTarPerm modified mode [ 'bin/unalias.js', 436, 438 ]
81 silly gunzTarPerm extractEntry bin/mkdir.js
82 silly gunzTarPerm modified mode [ 'bin/mkdir.js', 436, 438 ]
83 silly gunzTarPerm extractEntry bin/rebuilt.json
84 silly gunzTarPerm modified mode [ 'bin/rebuilt.json', 436, 438 ]
85 silly gunzTarPerm extractEntry commands.json
86 silly gunzTarPerm modified mode [ 'commands.json', 484, 502 ]
87 silly gunzTarPerm extractEntry dist/mute.js
88 silly gunzTarPerm modified mode [ 'dist/mute.js', 484, 502 ]
89 verbose write writing to C:\Users\user\AppData\Roaming\npm\node_modules\cash-global\package.json
90 info preinstall cash-global@0.1.0
91 verbose unsafe-perm in lifecycle true
92 info cash-global@0.1.0 Failed to exec preinstall script
93 verbose unlock done using C:\Users\user\AppData\Roaming\npm-cache\_locks\cash-global-c09d7c9a277aab3c.lock for C:\Users\user\AppData\Roaming\npm\node_modules\cash-global
94 verbose stack Error: cash-global@0.1.0 preinstall: `node ./bin/preinstall.js`
94 verbose stack Exit status 1
94 verbose stack     at EventEmitter.<anonymous> (C:\Program Files (x86)\nodejs\node_modules\npm\lib\utils\lifecycle.js:214:16)
94 verbose stack     at EventEmitter.emit (events.js:110:17)
94 verbose stack     at ChildProcess.<anonymous> (C:\Program Files (x86)\nodejs\node_modules\npm\lib\utils\spawn.js:24:14)
94 verbose stack     at ChildProcess.emit (events.js:110:17)
94 verbose stack     at maybeClose (child_process.js:1016:16)
94 verbose stack     at Process.ChildProcess._handle.onexit (child_process.js:1088:5)
95 verbose pkgid cash-global@0.1.0
96 verbose cwd C:\
97 error Windows_NT 6.1.7601
98 error argv "C:\\Program Files (x86)\\nodejs\\\\node.exe" "C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "cash-global" "-g"
99 error node v0.12.10
100 error npm  v2.14.9
101 error code ELIFECYCLE
102 error cash-global@0.1.0 preinstall: `node ./bin/preinstall.js`
102 error Exit status 1
103 error Failed at the cash-global@0.1.0 preinstall script 'node ./bin/preinstall.js'.
103 error This is most likely a problem with the cash-global package,
103 error not with npm itself.
103 error Tell the author that this fails on your system:
103 error     node ./bin/preinstall.js
103 error You can get their info via:
103 error     npm owner ls cash-global
103 error There is likely additional logging output above.
104 verbose exit [ 1, true ]
105 verbose unbuild node_modules\cash-global
106 info preuninstall cash-global@0.1.0
107 info uninstall cash-global@0.1.0
108 verbose unbuild rmStuff cash-global@0.1.0 from C:\Users\user\AppData\Roaming\npm\node_modules
109 silly gentlyRm C:\Users\user\AppData\Roaming\npm\$.cmd is being gently removed
110 silly gentlyRm verifying C:\Users\user\AppData\Roaming\npm is an npm working directory
111 silly gentlyRm C:\Users\user\AppData\Roaming\npm\cash.cmd is being gently removed
112 silly gentlyRm verifying C:\Users\user\AppData\Roaming\npm is an npm working directory
113 silly gentlyRm containing path C:\Users\user\AppData\Roaming\npm is under npm's control, in C:\Users\user\AppData\Roaming\npm
114 silly gentlyRm deletion target C:\Users\user\AppData\Roaming\npm\$.cmd is under C:\Users\user\AppData\Roaming\npm
115 verbose gentlyRm vacuuming from C:\Users\user\AppData\Roaming\npm\$.cmd up to C:\Users\user\AppData\Roaming\npm
116 silly gentlyRm containing path C:\Users\user\AppData\Roaming\npm is under npm's control, in C:\Users\user\AppData\Roaming\npm
117 silly gentlyRm deletion target C:\Users\user\AppData\Roaming\npm\cash.cmd is under C:\Users\user\AppData\Roaming\npm
118 verbose gentlyRm vacuuming from C:\Users\user\AppData\Roaming\npm\cash.cmd up to C:\Users\user\AppData\Roaming\npm
119 silly gentlyRm C:\Users\user\AppData\Roaming\npm\$ is being gently removed
120 silly gentlyRm verifying C:\Users\user\AppData\Roaming\npm is an npm working directory
121 silly gentlyRm containing path C:\Users\user\AppData\Roaming\npm is under npm's control, in C:\Users\user\AppData\Roaming\npm
122 silly gentlyRm deletion target C:\Users\user\AppData\Roaming\npm\$ is under C:\Users\user\AppData\Roaming\npm
123 verbose gentlyRm vacuuming from C:\Users\user\AppData\Roaming\npm\$ up to C:\Users\user\AppData\Roaming\npm
124 silly gentlyRm C:\Users\user\AppData\Roaming\npm\cash is being gently removed
125 silly gentlyRm verifying C:\Users\user\AppData\Roaming\npm is an npm working directory
126 silly gentlyRm containing path C:\Users\user\AppData\Roaming\npm is under npm's control, in C:\Users\user\AppData\Roaming\npm
127 silly gentlyRm deletion target C:\Users\user\AppData\Roaming\npm\cash is under C:\Users\user\AppData\Roaming\npm
128 verbose gentlyRm vacuuming from C:\Users\user\AppData\Roaming\npm\cash up to C:\Users\user\AppData\Roaming\npm
129 info postuninstall cash-global@0.1.0
130 silly gentlyRm C:\Users\user\AppData\Roaming\npm\node_modules\cash-global is being purged from base C:\Users\user\AppData\Roaming\npm
131 verbose gentlyRm don't care about contents; nuking C:\Users\user\AppData\Roaming\npm\node_modules\cash-global
132 silly vacuum-fs purging C:\Users\user\AppData\Roaming\npm\node_modules\cash-global
133 silly vacuum-fs quitting because other entries in C:\Users\user\AppData\Roaming\npm\node_modules
```

Do you know what is the problem?

**Comment by tontonrally** [comment]
Same here on Windows 10

**Comment by dthree** [comment]
Shoot it looks like a compat problem with node < 4.  Will fix asap, but try upgrading Node.

**Comment by Jonarzz** [comment]
Upgrading Node fixed it, thanks a lot!

**Comment by dthree** [comment]
Great!
Keeping issue open until I fix.

**Comment by dthree** [comment]
This is fixed in `v0.1.1` of `cash-global`.
